core/src/test/scala/integration/kafka/api/BaseProducerSendTest.scala

```diff
@@ -224,7 +224,7 @@ abstract class BaseProducerSendTest extends KafkaServerTestHarness {
         val record = new ProducerRecord[Array[Byte], Array[Byte]](topic, partition, baseTimestamp + i, "key".getBytes, "value".getBytes)
         producer.send(record, callback)
       }
-      producer.close(10000L, TimeUnit.MILLISECONDS)
+      producer.close(20000L, TimeUnit.MILLISECONDS)
```

Comment:
By the way, this used to be 5000L and I doubled it to 10000L in May. Seems like that wasn't enough.